spec/classes/lightblue/application/datamgmt_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe 'lightblue::application::datamgmt' do
+  let(:hiera_config){ 'spec/fixtures/hiera/hiera.yaml' }
+
+  let :facts do
+    {
+      :architecture => 'x86_64',
+      :osfamily => 'RedHat',
+      :operatingsystemrelease => '7.0'
+    }
+  end
+
+  context 'with default params' do
+    it do
+      should contain_package('lightblue-data-mgmt') \
+        .with_ensure('latest') \
+        .with_require(['Class[Lightblue::Yumrepo::Lightblue]', 'Class[Lightblue::Eap]'])
+    end
+
+    it do
```

Comment:
I know, I just generally prefer having many smaller tests to fewer larger tests. One assertion per test is ideal IMO. I guess you could argue condensing it doesn't hurt since rspec gives you nice test failure output anyway. I suppose one disadvantage would be, if a single test has many failed assertions, you'd only see the first one, however if they were separate `it` blocks, you would see all of them together.